.github/workflows/deploy_to_firebase_app_distribution.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy staging and production to Firebase App Distribution
+
+on:
+  # Trigger the workflow on push action in main branch.
+  # So it will trigger when the PR of the release branch was merged.
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy_staging_and_production_to_firebase_app_distribution:
+    name: Deploy staging and production to Firebase App Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Set up timezone
+        uses: zcong1993/setup-timezone@master
+        with:
+          timezone: Asia/Bangkok
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Run Detekt
+        run: ./gradlew detekt
+
+      - name: Archive Detekt reports
+        uses: actions/upload-artifact@v2
+        with:
+          name: DetektReports
+          path: build/reports/detekt/
+
+      - name: Run unit tests and Jacoco
+        run: ./gradlew jacocoTestReport
+
+      - name: Archive code coverage reports
+        uses: actions/upload-artifact@v2
+        with:
+          name: CodeCoverageReports
+          path: |
+            app/build/reports/jacoco/jacocoTestReport/
+            data/build/reports/jacoco/jacocoTestReport/
+
+      - name: Build debug APK
+        run: ./gradlew assembleDebug
+
+      - name: Deploy staging to Firebase
+        uses: wzieba/Firebase-Distribution-Github-Action@v1
+        with:
+          appId: ${{secrets.FIREBASE_APP_ID_STAGING}}
+          token: ${{secrets.FIREBASE_TOKEN}}
```

Comment:
Have we configured firebase app yet? I can not find these secrets at https://github.com/nimblehq/jetpack-compose-crypto/settings/secrets/actions 🤔

Reply:
@luongvo Added 🚀

Reply:
@luongvo Added 🚀